resources/assets/app.scss

```diff
@@ -27,3 +27,7 @@ button.delete {
   padding: $section-spacing 0;
   border-bottom: 2px solid $purple;
 }
+
+.figure__media img {
```

Comment:
Just want to make sure we're not using this anywhere else in this app, right?

Reply:
It might be a good idea to scope this to a `.rounded-figure` class just in case.

```scss
.figure-rounded .figure__media img {
 // ...
```

Reply:
oh good call! @DFurnes looks like we're using it here: https://github.com/DoSomething/rogue/search?utf8=%E2%9C%93&q=figure__media&type=

But doesn't have an `img`. Just in case, should I add a `staff_pic` class to this too?

Reply:
Yeah, whatever class you want to use to scope the styles is fine. 👍

Reply:
updated :) is this cool to merge now @DFurnes ?